plugins/modules/accesspoint_workflow_manager.py

```diff
@@ -3193,6 +3355,27 @@ def main():
     else:
         ccc_network.log("No reboot list found in validated configuration.", "WARNING")
```

Comment:
Why do we need to print warning message to the user?

Reply:
removed the else part